contracts/src/v0.8/ccip/FeeQuoter.sol

```diff
@@ -1164,6 +1179,7 @@ contract FeeQuoter is AuthorizedCallers, IFeeQuoter, ITypeAndVersion, IReceiver,
           || (
```

Comment:
nit: on line 1175 the comment should read "supported chain family is EVM, SVM, and APTOS